Directory.Packages.props

```diff
@@ -180,6 +179,7 @@
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="$(MicrosoftExtensionsDiagnosticsHealthChecksEntityFrameworkCoreLTSVersion)" />
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="$(MicrosoftExtensionsDiagnosticsHealthChecksLTSVersion)" />
     <PackageVersion Include="Microsoft.Extensions.Features" Version="$(MicrosoftExtensionsFeaturesLTSVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.FileProviders.Embedded" Version="$(MicrosoftExtensionsFileProvidersEmbeddedLTSVersion)" />
```

Comment:
should we be updating this? it's in the group per the comment on line 155 that arcade should be updating for us? cc @joperezr

Reply:
This nuget package comes from the dotnet/aspnetcore repo. It was misplaced previously. So this change is moving it to be co-located and consistent with the rest of the dotnet/aspnetcore packages.